### PR TITLE
plugin/forward: clarify relation with proxy

### PR DESCRIPTION
**Pull request by miekg**
some other small bits as well.

### CODE REVIEW

File: plugin/forward/README.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ connect to a random upstream (which may or may not work).
 
 This plugin can only be used once per Server Block.
 
+How does *forward* relate to *proxy*? This plugin is the "new" version of *proxy* and is faster
+because it re-uses connections to the upstreams. It also does in-band health checks - using DNS
+instead of HTTP. Since it is newer it has a little less (production) millage on it.
```
Comment (johnbelamaric) [inline plugin/forward/README.md:26]: s/millage/milage/

Comment (chrisohaver) [reply plugin/forward/README.md:26]: https://trends.google.com/trends/explore?q=milage,mileage

Comment (yuewko) [reply plugin/forward/README.md:26]: Very persuasive!

**Comment by fturib** [comment]
@miekg:
About Proxy vs Forward, I noticed that:
- proxy support gRPC protocol but NOT TLS
- forward support TLS but not gRPC

is that correct ?

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] plugin/forwar..." ]
>@miekg:
>About Proxy vs Forward, I noticed that:
>- proxy support gRPC protocol but NOT TLS
>- forward support TLS but not gRPC
>
>is that correct ?

Yep. allthough proxy tls support should not be that hard, grpc in forward is 
harder.

**Pull request merged**